src/main/java/org/lanternpowered/server/data/manipulator/DataManipulatorRegistry.java

```diff
@@ -679,10 +679,12 @@ public static DataManipulatorRegistry get() {
                 c -> c.registerKey(Keys.IS_FLYING, false));
         register(FoodData.class, ImmutableFoodData.class,
                 c -> {
-                    c.registerKey(Keys.FOOD_LEVEL, 20, 0, LanternKeys.MAX_FOOD_LEVEL);
                     c.registerKey(LanternKeys.MAX_FOOD_LEVEL, 20, 0, Integer.MAX_VALUE);
-                    c.registerKey(Keys.EXHAUSTION, 0.0, 0.0, 40.0);
-                    c.registerKey(Keys.SATURATION, 0.0, 0.0, 5.0);
+                    c.registerKey(Keys.FOOD_LEVEL, 20, 0, LanternKeys.MAX_FOOD_LEVEL);
+                    c.registerKey(LanternKeys.MAX_EXHAUSTION, 40.0, 0.0, Double.MAX_VALUE);
+                    c.registerKey(Keys.EXHAUSTION, 0.0, 0.0, LanternKeys.MAX_EXHAUSTION);
+                    c.registerKey(LanternKeys.MAX_SATURATION, 40.0, 0.0, Double.MAX_VALUE);
```

Comment:
It seems that's the default saturation. I can't find a clear max, as it seems to actually change?

"The maximum value of the food saturation level is equal to the current value of your food level"